bin/dev-tools/create_shipments.py

```diff
@@ -0,0 +1,332 @@
+            if not moderator_response:
+                if self.verbose:
+                    print(f'Error generating moderator wallet.')
+                self._process_failure(f'Error generating moderator wallets')
+
+            self.attributes['moderator_wallet_id'] = moderator_response['data']['id']
+        self.attributes['shipper_wallet_id'] = shipper_response['data']['id']
+        self.attributes['carrier_wallet_id'] = carrier_response['data']['id']
+
+    def _set_storage_credentials(self):
+        if self.verbose:
+            print(f'Creating storage credentials.')
+
+        response = self._parse_request(
+            f"{self.profiles_url}/api/v1/storage_credentials", data={
+                'driver_type': 'local',
+                'base_path': '/shipments',
+                'options': "{}",
+                'title': f'Shipment creator SC: {str(uuid4())}'
+            }, headers={'Authorization': 'JWT {}'.format(self.shipper['token'])})
+
+        if not response:
+            self._process_failure(f'Error generating storage credentials')
+
+        if self.verbose:
+            print(f'Created storage credentials: {response["data"]["id"]}')
+        self.attributes['storage_credentials_id'] = response['data']['id']
+
+    # pylint:disable=unused-variable
+    def _add_telemetry(self, device_id):
+        if self.verbose:
+            print(f'Creating sensors for device: {device_id}')
+        telemetry_data = []
+        self.telemetry_data['device_id'] = device_id
+        for i in range(3):
+            attributes = {
+                'name': f'Sensor: {str(uuid4())}',
+                'hardware_id': f'Hardware_id: {str(uuid4())}',
+                'units': 'c',
+            }
+            response = self._parse_request(
+                f'{self.profiles_url}/api/v1/device/{device_id}/sensor/',
+                data=attributes,
+                headers={'Authorization': 'JWT {}'.format(self.shipper['token'])}
+            )
+            if not response:
+                print(f'{"Failed to create" if not response.ok else "created"} sensor for device: {device_id}')
+                continue
+
+            telemetry_copy = deepcopy(self.telemetry_data)
+            telemetry_copy['hardware_id'] = attributes['hardware_id']
+            telemetry_copy['sensor_id'] = response['data']['id']
+            for j in range(3):
+                telemetry_copy['value'] = randint(0, 100)
+                telemetry_data.append({"payload": telemetry_copy})
+        if self.verbose:
+            print(f'Adding telemetry via device: {device_id}')
+        telemetry_response = requests.post(
+            f"{self.transmission_url}/api/v1/devices/{device_id}/telemetry",
+            json=telemetry_data,
+            headers={"Content-type": "application/json"}
+        )
+        if self.verbose:
+            print(f'{"Failed to add" if not telemetry_response.ok else "Added"} tracking via device: {device_id}')
```

Comment:
These "double duty" print statements might cause you headaches in the future.  I recommend you check for the error case explicitly and log that with a level of WARNING (or higher depending on how you handle the error) and then log the success as a DEBUG level message separately.